SQF/dayz_code/Configs/CfgServerTrader/Category/FriendlyAssaultRifle.hpp

```diff
@@ -479,17 +479,133 @@ class Category_485 {
+		sell[] = {6,"ItemGoldBar"};
+	};
+	class L85A2_SD_FL_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {7,"ItemGoldBar"};
+	};
+	class L85A2_SD_MFL_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {8,"ItemGoldBar"};
+	};
+	class L85A2_CCO_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {4,"ItemGoldBar"};
+	};
+	class L85A2_CCO_FL_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {5,"ItemGoldBar"};
+	};
+	class L85A2_CCO_MFL_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {6,"ItemGoldBar"};
+	};
+	class L85A2_CCO_SD_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {8,"ItemGoldBar"};
+	};
+	class L85A2_CCO_SD_FL_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {9,"ItemGoldBar"};
+	};
+	class L85A2_CCO_SD_MFL_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {1,"ItemGoldBar10oz"};
+	};
+	class L85A2_Holo_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {4,"ItemGoldBar"};
+	};
+	class L85A2_Holo_FL_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {5,"ItemGoldBar"};
+	};
+	class L85A2_Holo_MFL_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {6,"ItemGoldBar"};
+	};
+	class L85A2_Holo_SD_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {8,"ItemGoldBar"};
+	};
+	class L85A2_Holo_SD_FL_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {9,"ItemGoldBar"};
+	};
+	class L85A2_Holo_SD_MFL_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {1,"ItemGoldBar10oz"};
+	};
+	class L85A2_ACOG_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {6,"ItemGoldBar"};
+	};
+	class L85A2_ACOG_FL_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {7,"ItemGoldBar"};
+	};
+	class L85A2_ACOG_MFL_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {8,"ItemGoldBar"};
+	};
+	class L85A2_ACOG_SD_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {1,"ItemGoldBar10oz"};
+	};
+	class L85A2_ACOG_SD_FL_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {11,"ItemGoldBar"};
+	};
+	class L85A2_ACOG_SD_MFL_DZ {
+		type = "trade_weapons";
+		buy[] = {-1,"ItemGoldBar"}; //Sell only
+		sell[] = {12,"ItemGoldBar"};
+	};
+	class BAF_L85A2_RIS_SUSAT { //Not sure if this should be kept in 1.0.6.2.
 		type = "trade_weapons";
 		buy[] = {6,"ItemGoldBar"};
 		sell[] = {3,"ItemGoldBar"};
 	};
-	class BAF_L85A2_RIS_ACOG {
+	class BAF_L85A2_RIS_ACOG { //Streatman's model L85A2_ACOG_DZ should replace this.

```

Comment:
You can take out `BAF_L85A2_RIS_SUSAT` and `BAF_L85A2_RIS_ACOG`, just remove them from the loot tables and add an update to 1.0.6.2 sql to swap them out for the closest equivalent.